### DISCUSSION TITLE
1.5.0-beta5

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#381] [Обновления]
> [!WARNING]
> Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений
- Фикс меню выбора языка и слайдеров для YouTube в Firefox (#371)
- Переработан дропдаун, теперь, он соответствует стилю расширения, а так же, теперь, в нем можно легко находить языки с помощью поиска
- Теперь, если сервер определения языка текста не отвечает будет установлен английский язык (#361) от @SashaXser в #366
- функция проксирование аудио, теперь, по умолчанию включена, если вы используете cloudflare версию и ваш язык браузера украинский от @SashaXser в #366
- Убрана проверка на премьеру для YouTube от @SashaXser в #366

## What's Changed
* Удаление использования isPremiere by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/366

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0-beta4...1.5.0-beta5

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0-beta5'>1.5.0-beta5</a>.</em>

**Comment by NullVerdict** [comment]
Существует проблема, связанная с переводом стримов. Запросы на сайт прокси не прекращаются даже после остановки перевода или закрытия стрима. Это значит, что сайт прокси продолжает получать ненужный трафик.

**Reply by ilyhalight** [reply]
если не забуду, то постараюсь исправить

**Comment by SrevSA** [comment]
К сожалению сервера Яндекс бастуют и перевод на украинский не решен. Очень жаль. Решение было идеальным. Я только могу поблагодарить вас за проделанную работу, но увы ,моя проблема не решена. Я о последней
вашей Бете сейчас. Возможно есть какие то варианты? Звуковую дорожку на русском я могу извлечь,а вот дальше ступор.. Не понимаю как аудио перевести .

**Reply by NullVerdict** [reply]
> К сожалению сервера Яндекс бастуют и перевод на украинский не решен. Очень жаль. Решение было идеальным. Я только могу поблагодарить вас за проделанную работу, но увы ,моя проблема не решена. Я о последней вашей Бете сейчас. Возможно есть какие то варианты? Звуковую дорожку на русском я могу извлечь,а вот дальше ступор.. Не понимаю как аудио перевести .

Свяжитесь со службой поддержки Яндекса, они смогут предоставить вам точную информацию о причинах отключения украинского языка.

**Reply by SrevSA** [reply]
SashaXser, у вас предвзятое отношение или вы опять не выспались? Я обязательно последую вашему совету, но позже.  Наверное вы догадываетесь когда.. С уважением к вашему труду.

**Reply by ilyhalight** [reply]
> К сожалению сервера Яндекс бастуют и перевод на украинский не решен. Очень жаль. Решение было идеальным. Я только могу поблагодарить вас за проделанную работу, но увы ,моя проблема не решена. Я о последней вашей Бете сейчас. Возможно есть какие то варианты? Звуковую дорожку на русском я могу извлечь,а вот дальше ступор.. Не понимаю как аудио перевести .

Это проблема на стороне яндекса, мы тут ничего не можем сделать

**Comment by ACherepkov1989** [comment]
Окно настроек не полностью переведено, хотя перевод строк в репозитории имеется, но по непонятной причине перевод полностью не подгружаются в расширение. Проверено на ноутбуке и настольном пк. 

![Снимок экрана 2023-11-01 132012](https://github.com/ilyhalight/voice-over-translation/assets/4404804/4f279a8a-6ae3-48f9-9ff1-53fdd052807f)
![Снимок экрана 2023-11-01 132112](https://github.com/ilyhalight/voice-over-translation/assets/4404804/7b09f800-eacb-42de-a727-c5bae0245d4b)

**Reply by ilyhalight** [reply]
Переводы берутся из релизной версии, а там еще этой локализации нету

**Reply by ACherepkov1989** [reply]
Понял, спс.

**Reply by ilyhalight** [reply]
в тестовом режиме сделал подгрузку актуальных локализаций в бете

**Comment by ACherepkov1989** [comment]
Если открыть видео через vk.com, то расширение на английском, а если через vk.ru, то на русском. Эта проблема также и на версии 1.4.1.

**Reply by ilyhalight** [reply]
возможно исправлено в последней бете
